components/modal/modal.css

```diff
@@ -0,0 +1,39 @@
+.modal-container{
```

Comment:
Suggestion (do later):
Make background non-scrollable when modal is shown

Else everything looks good!
Good to merge!